maven-core/src/test/java/org/apache/maven/project/ProjectBuilderTest.java

```diff
@@ -119,6 +119,18 @@ public void testResolveDependencies()
         assertEquals( 1, results.size() );
         MavenProject mavenProject = results.get( 0 ).getProject();
         assertEquals( 1, mavenProject.getArtifacts().size() );
+
+        final MavenProject project = mavenProject;
+        final int[] getArtifactsResultInAnotherThead = { 0 };
```

Comment:
Youi'll likely want to use an AtomicInteger here instead of this classic hack (in the good sense of the word hack of course)